README.md

```diff
@@ -1,6 +1,28 @@
 # cog-triton
 A cog implementation of Nvidia's Triton server
 
+## Error codes
+
+Universal user errors:
+
+E000: Generic user error (not used)
```

Comment:
You'll want to add bullets (`*` or `-`) to these lines, otherwise Markdown will pull them all onto one line when it renders: 
![image](https://github.com/replicate/cog-triton/assets/1493712/eb0a7e61-39fe-4761-96f1-4d1b9c9f4c6f).